tests/test_proto.py

```diff
@@ -0,0 +1,10 @@
+def test_imports():
+    from substrait.proto.pysubstrait.algebra_pb2 import Expression
```

Comment:
What should the protobuf class namespace be in the package? `pysubstrait`? `python`? `python.substrait`? etc.

Reply:
i think pysubstrait -> substrait